braindecode/training/losses.py

```diff
@@ -19,3 +19,48 @@ def forward(self, preds, targets):
         avg_preds = torch.mean(preds, dim=2)
         avg_preds = avg_preds.squeeze(dim=1)
         return self.loss_function(avg_preds, targets)
+
+
+class MixupCriterion:
+    """Implements loss for Mixup for EEG data. See [1]_.
+
+    Implementation based on [2]_.
+
+    Parameters
+    ----------
+    preds : torch.Tensor
+        Predictions from the model.
+    target : torch.Tensor | list of torch.Tensor
+        For predictions without mixup, the targets as a tensor. If mixup has
+        been applied, a list containing the targets of the two mixed
+        samples and the mixing coefficients as tensors.
```

Comment:
This seems confusing to me that this is under Parameters. Parameters I would expect it is parameters for constructor of the class, not the arguments of the `__call__` function. This is also like that in https://pytorch.org/docs/stable/generated/torch.nn.BCEWithLogitsLoss.html where the parameters are really class constructor arguments. Canw e somehow call this differently? or move parameter part doc to the loss_function?

Reply:
Ping @sbbrandt ☝️

Reply:
Can we may be just fix it together tomorrow? @sbbrandt  might be busy with master thesis @cedricrommel

Reply:
and this is only thing keeping me from merge atm

Reply:
Yes, I agree. The parameter doc should probably be moved to the static loss function.

Reply:
I made this a plain function but I realize this is not test for this...

@sbbrandt can you push one test for mixup_criterion ?

Reply:
I implemented a basic test for the 'mixup_criterion'.

Should there be added anything to the test @agramfort ?